onnxruntime/core/framework/fuse_nodes_funcs.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include "core/common/common.h"
+#include "core/framework/execution_provider.h"
+#include "core/framework/ex_lib_loader.h"
+
+namespace onnxruntime {
+
+class FuseFuncManager {
```

Comment:
Should it be renamed to FuncManager which will be more general? In future, it can be extended for the customized kernel which is not for fused operation.